From edf6351fc0a4f8babcbfd61dfdd8d109537e96d9 Mon Sep 17 00:00:00 2001
From: honjow <honjow311@gmail.com>
Date: Wed, 30 Apr 2025 22:19:34 +0800
Subject: [PATCH 2/2] Add support for more handheld devices to fan detection

---
 src/overlay.cpp | 11 ++++++++++-
 1 file changed, 10 insertions(+), 1 deletion(-)

diff --git a/src/overlay.cpp b/src/overlay.cpp
index 4e0d4d9..5125e4c 100644
--- a/src/overlay.cpp
+++ b/src/overlay.cpp
@@ -868,7 +868,16 @@ void update_fan(){
       auto dirs = ls(path.c_str(), "hwmon", LS_DIRS);
       for (auto& dir : dirs) {
          string full_path = (path + dir + "/name").c_str();
-         if (read_line(full_path).find("steamdeck_hwmon") != string::npos){
+         string name = read_line(full_path);
+         if (name.find("steamdeck_hwmon") != string::npos ||
+             name.find("asus") != string::npos ||
+             name.find("aynec") != string::npos ||
+             name.find("gpdfan") != string::npos ||
+             name.find("msi_ec") != string::npos ||
+             name.find("msi-ec") != string::npos ||
+             name.find("msi_wmi_platform") != string::npos ||
+             name.find("oxp_ec") != string::npos ||
+             name.find("oxpec") != string::npos){
             hwmon_path = path + dir + "/fan1_input";
             break;
          }
-- 
2.49.0

